traffic_ops/traffic_ops_golang/origin/origins.go

```diff
@@ -487,3 +488,217 @@ func deleteQuery() string {
 WHERE id=:id`
 	return query
 }
+
+// Get is the handler for GET requests to Origins of APIv5
+func Get(w http.ResponseWriter, r *http.Request) {
+
+	var useIMS bool
+
+	inf, sysErr, userErr, errCode := api.NewInfo(r, nil, nil)
+	if sysErr != nil || userErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+
+	origins, userErr, sysErr, errCode, maxTime := getOrigins(w.Header(), inf.Params, inf.Tx, inf.User, useIMS)
+	var returnable []tc.OriginV5
```

Comment:
If zero origins match the filtering parameters sent by the client, this will cause it to return a `null` `response` property, when the proper representation for an empty list is the empty array, i.e. `[]`.

Reply:
You don't need to do anything here if there are no origins,  the response will be an empty array by default